draft-ietf-emu-rfc7170bis.md

```diff
@@ -446,27 +446,52 @@ allows support for protection of the peer's identity when using TLS
-certificate and SHOULD also examine the EAP server name presented in
-the certificate in order to determine whether the EAP server can be
-trusted.  When performing server certificate validation,
-implementations MUST provide support for the rules in {{RFC5280}} for
-validating certificates against a known trust anchor.  In addition,
-implementations MUST support matching the realm portion of the peer's
-NAI against a SubjectAltName of type dNSName within the server
-certificate.  However, in certain deployments, this might not be
-turned on.  Please note that in the case where the EAP authentication
+the peer.
+This section describes the validation that the Peer SHOULD do in order to accept the certificate.
+There are a number of situations where the Peer does not do validation:
+
+* when the Peer has the Server's End Entity (EE) certificate pinned or loaded directly into it's Trusted Anchor store {{?RFC4949}}.
+
+* when the Peer is certain that it will be using an authenticated Inner method
+
+Situations in which the Peer will be doing some kind of onboarding operation ("bootstrapping") may require that the Peer defer the verification of the digital signature on the server's certificate, however the content verification described below MUST still be performed.
+
+For the server certificate validation, implementations MUST provide support for the rules in {{RFC5280}} for validating certificates against a known trust anchor.
+
+The Peer MUST verify the validity of the EAP server certificate and MUST also examine the EAP server name presented in the certificate in order to determine whether the EAP server can be
+trusted.
+
+The Peer MUST do DNS-ID style verification of the Realm portion of the Peer's NAI.
+{{!ID.ietf-uta-rfc6125bis, Section 6}} describes the mechanism in detail, and this document notes that comparison against the SubjectDN is no longer indicated.
+This Realm will often also have been provided in a configuration, but there are cases where a Peer device is able to pick a service from a list of available realms.
+
+The use of Extended Key Usage attributes in server certificates is OPTIONAL, and in may deployments it may be necessary for the Peer to be able to omit verification of them.
+
+(MORE DISCUSSION NEEDED HERE)
+Please note that in the case where the EAP authentication
 is remote, the EAP server will not reside on the same machine as the
 authenticator, and therefore, the name in the EAP server's
 certificate cannot be expected to match that of the intended
-destination.  In this case, a more appropriate test might be whether
-the EAP server's certificate is signed by a certification authority
-(CA) controlling the intended domain and whether the authenticator
+destination.
+
+In this case, a more appropriate test might be whether the EAP server's certificate is signed by a certification authority (CA) controlling the intended domain and whether the authenticator
```

Comment:
This puts the EAP certs into a the web PKI.  And also requires domain checking, which can't be done until the peer has network access.

I think the solution here is again to suggest correlation between the EAP Identity / NAI, and the server name.